### ISSUE TITLE
1.10.4

### ISSUE THREAD

**Issue by wgalafassijr** [StrobeTechnologiesLtd/OSTCheckList#4]
Hi,

This plugin work with version 1.10.4?

Regards,
Wilson

**Comment by robintoy** [comment]
It should do in theory, but I don't think is has been tested for this yet.

**Comment by wgalafassijr** [comment]
can you confirm the install instructions please?

i have to copy:

include to include/plugins
scp to scp
and the other 3 (checkist.php/config.php/plugin.php) files to root folder?

i have to run something manually like the mysql included on install folder? if not where i have to copy that folder?

regards,
wilson

**Comment by robintoy** [comment]
Hi Wilson,

First of all we have a listed bug #2 with the installer and the SQL creating the required tables. To get around this please run the following SQL: -

```SQL
CREATE TABLE IF NOT EXISTS `ost_checklist` (
  `id` bigint(20) unsigned NOT NULL auto_increment,
  `orde` int(11) NOT NULL,
  `menu_id` int(11) NOT NULL default '0',
  `indent` int(11) NOT NULL default '0',
  `header` int(11) NOT NULL default '0',
  `period` int(11) NOT NULL default '0',
  `tekst` varchar(250) default NULL,
  `disabled` int(11) NOT NULL default '0',
  `start` date NOT NULL,
  `help` text NOT NULL,
  PRIMARY KEY  (`id`),
  UNIQUE KEY `id` (`id`),
  KEY `orde` (`orde`)
) ENGINE=MyISAM  DEFAULT CHARSET=latin1 AUTO_INCREMENT=1 ;

CREATE TABLE IF NOT EXISTS `ost_checklist_entries` (
  `id` bigint(20) unsigned NOT NULL auto_increment,
  `datum` datetime default NULL,
  `door` varchar(30) default NULL,
  `ref` int(11) default NULL,
  `tekst` text,
  `status` int(11) NOT NULL default '0',
  PRIMARY KEY  (`id`),
  UNIQUE KEY `id` (`id`),
  KEY `datum` (`datum`),
  KEY `ref` (`ref`),
  FULLTEXT KEY `tekst` (`tekst`),
  FULLTEXT KEY `volletekst` (`tekst`,`door`)
) ENGINE=MyISAM  DEFAULT CHARSET=latin1 AUTO_INCREMENT=1 ;
```

The readme then says we need to do the following: -

1. Copy contents to your OSTICKET_ROOT/include/plugins directory
2. Copy the sub folder SCP to OSTICKET_ROOT/scp directory
3. Login to osTicket as your admin account and enable plugin
4. If you would like the stats to work please install JpGraph from http://jpgraph.net/ to scp/jpgraph/src directory

But if I expand on this a little with details it would end up being: -

1. Copy "check-list" to your OSTICKET_ROOT/include/plugins/ directory
2. Copy the sub folder SCP from "check-list" to OSTICKET_ROOT/scp/ directory
3. Login to osTicket as your admin account and enable plugin
4. If you would like the stats to work please install JpGraph from http://jpgraph.net/ to scp/jpgraph/src directory


Hope this helps.

Robin

**Comment by wgalafassijr** [comment]
After doing that i have error 500 on my installation.

**Comment by robintoy** [comment]
OK,

looks like this might not have been upgraded.
We have not looked at this mod in years as discussed in our other bug listings.

**Comment by wgalafassijr** [comment]
Yes. I perfectly understand. If some time you update this i will give i try again. 

Anyway thank you very much.

Regards,
Wilson

**Comment by zeHamish** [comment]
I must be holding my tongue incorrectly - the addin appears to be loaded ok, i get the option under APplications but New item doesn't produce an expected entry dialogue - behaves like a page refresh

just because there was some ambiguity about installation files, locations et al...i yanked it, read everything and redid it a couple of times.

----
OST 1.10.4
PHP 7.0.31 (arbitrarily held down for osT)
osTicket_v1-10-xx_TimeTracking installed (i have an output report issue)

check-list 0.2.0
plugin enabled for admin and staff
SQL executed - manual with appropriate prefix and terminator
SCP folder content copied over
JPgraph src plugged in (sooks because of no data - neh)

i have file system objects...
.../include/plugin/check-list/
checklist.php
config.php
plugin.php
.../include/plugin/check-list/include/
calendar.php
class.checklist-installer.php
lang_uk_en.php
lib.php

**Comment by wgalafassijr** [comment]
> 
> 
> I must be holding my tongue incorrectly - the addin appears to be loaded ok, i get the option under APplications but New item doesn't produce an expected entry dialogue - behaves like a page refresh
> 
> just because there was some ambiguity about installation files, locations et al...i yanked it, read everything and redid it a couple of times.
> 
> OST 1.10.4
> PHP 7.0.31 (arbitrarily held down for osT)
> osTicket_v1-10-xx_TimeTracking installed (i have an output report issue)
> 
> check-list 0.2.0
> plugin enabled for admin and staff
> SQL executed - manual with appropriate prefix and terminator
> SCP folder content copied over
> JPgraph src plugged in (sooks because of no data - neh)
> 
> i have file system objects...
> .../include/plugin/check-list/
> checklist.php
> config.php
> plugin.php
> .../include/plugin/check-list/include/
> calendar.php
> class.checklist-installer.php
> lang_uk_en.php
> lib.php

in this way worked fine for you?

**Comment by zeHamish** [comment]
negative.

i can't add tasks, the add task link behaves like a page refresh.

**Comment by alexinc2000** [comment]
Hi. First of all,I found mistake on file: checklist.php in fuctions createStaffMenu, createAdminMenu. Search include file i didn't find class Application. Apache said that class Application don't find. 
OST 1.10.4
PHP 5.6.39 
osTicket_v1-10-xx_TimeTracking installed

check-list 0.2.0
plugin enabled for admin and staff.

Q: Where is it class?
